src/ui/ColorPicker.scss

```diff
@@ -0,0 +1,113 @@
+.widget-color-picker {
+    .sketch-picker, .compact-picker {
+        input {
+            width: 100% !important;
```

Comment:
Prefer if possible, it without `!important`? Can we use more specific selectors?

Reply:
The component already has an inline styling